scripts/just.config.ts

```diff
@@ -110,7 +128,7 @@ export function preset() {
   task('update-api', series('clean', 'copy', 'sass', 'ts', 'api-extractor:update'));
 
   task('dev:storybook', series('storybook:start'));
-  task('dev', series('copy', 'sass', 'webpack-dev-server'));
+  task('dev', series('copy', 'sass', webpackDevServer(getJustArgv())));
```

Comment:
I think it would be better to pass the args to the initial definition of the `webpack-dev-server` task.

Reply:
I'm afraid I don't follow ? can you elaborate please?

Reply:
Ah you meant -> `task('webpack-dev-server', webpackDevServer(getJustArgv()));` right? good call. thx

Reply:
Yup that's it :)